docs/root/start/sandboxes/opentelemetry.rst

```diff
@@ -112,6 +113,81 @@ In the ``Latency Samples`` of ``opentelemetry.proto.collector.trace.v1.TraceServ
 
 .. image:: /start/sandboxes/_static/opentelemetry-ui-traces.png
 
+Step 5: View the access logs in the OpenTelemetry collector output
+******************************************************************
+
+Check the spans Docker-compose logs:
+
+.. code-block:: console
+   :linenos:
```

Comment:
lets remove the linenos from the console dump - its meant more for files so users can see that its coming from a file and match the lines

Reply:
Done in b4a96fe0f5875d108a49b54fa30720273c35dcb8